### PR TITLE
**DO NOT MERGE** Standardized Codebase

### PR DESCRIPTION
**Pull request by riley-thomps**
DO NOT MERGE this PR until after @Babacar02 changes are complete so I can merge and deal with those conflicts here.

This PR standardizes our codebase in a concise, easy-to-read way. I followed a simple pattern for all files and reorganized where needed. The following things that have been standardized moving forward:

```
- Using brackets '{' for conditional statements
- The use of " vs. ' vs. `
- Unnecessary whitespace
- Reduce use of 'any' keyword on types with use of Interfaces
- Moved 'Depreciated/Unused' methods to bottom of file and removed their 'public' or 'export' access modifiers to 'private'
      - In the future we can decide if we should keep these unused methods or remove them for simplicity
- Moved variables scoped to a class/namespace to the top regardless if public/private
- Moved 'Private Helper Methods' to the bottom below the 'public' or 'export' methods, and above the 'Depreciated' methods
- Standardized import paths to start at 'src'
- Updated documentation to remove filler words and fill in any missing gaps
- Refactored some pieces of code to incorporate Interface logic and improve code readability
- Standardized use of semicolon ;
- Removed redundant/unused CSS
- Fixed any indentation errors
- Added necessary comments
- Other small changes
```

Main focus is to continue this standardized formatting moving forward so we can have a consistent readable codebase.

### CODE REVIEW

**Comment by CubeToPlay** [comment]
Do you think we should remove the gaps in the documentation/descriptions, I think it helps with the reading.

**Comment by riley-thomps** [comment]
> Do you think we should remove the gaps in the documentation/descriptions, I think it helps with the reading.

@CubeToPlay Is there a specific example you're referring to in the code so I can see where?

**Review by CubeToPlay — APPROVED** [review]
Everything Looks Good, the comments are weird, but we can keep this convention

File: src/frontend/Pages/Map/elements/abstracts/InfoWindowElement.ts [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
+    public readonly infoWindow: InfoWindow | undefined;
 
     /**
      * Constructor for the InfoElement class
-     * @param id        ID of the element
-     * @param map       map the element appears on
-     * @param marker    marker the element represents
+     * @param id ID of element
+     * @param map map the element appears on
+     * @param marker marker the element represents
```
[the remark this answers is not in the document]
Comment (riley-thomps) [reply src/frontend/Pages/Map/elements/abstracts/InfoWindowElement.ts:15]: I think overall it does help with readability, but the descriptions in the documentation is only a few words long. So although it helps its pretty negligible and uses a lot of whitespace to line-up all the params

**Pull request merged**